### ISSUE TITLE
FC from user reports - StatusView.java:36

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#165]
```
Taken from submitted user reports through Android Market. This is a FC that has 
happened on some users' devices.

v0.6.7
Feb 23, 2011 1:59:31 AM
2 reports/week
0%
42 reports

java.lang.RuntimeException: Unable to start activity 
ComponentInfo{com.gpl.rpg.AndorsTrail/com.gpl.rpg.AndorsTrail.activity.MainActiv
ity}: android.view.InflateException: Binary XML file line #8: Error inflating 
class com.gpl.rpg.AndorsTrail.view.StatusView
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2663)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2679)
at android.app.ActivityThread.access$2300(ActivityThread.java:125)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2033)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:123)
at android.app.ActivityThread.main(ActivityThread.java:4627)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:521)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:858)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
at dalvik.system.NativeStart.main(Native Method)
Caused by: android.view.InflateException: Binary XML file line #8: Error 
inflating class com.gpl.rpg.AndorsTrail.view.StatusView
at android.view.LayoutInflater.createView(LayoutInflater.java:513)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:565)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:618)
at android.view.LayoutInflater.inflate(LayoutInflater.java:407)
at android.view.LayoutInflater.inflate(LayoutInflater.java:320)
at android.view.LayoutInflater.inflate(LayoutInflater.java:276)
at 
com.android.internal.policy.impl.PhoneWindow.setContentView(PhoneWindow.java:198
)
at android.app.Activity.setContentView(Activity.java:1647)
at com.gpl.rpg.AndorsTrail.activity.MainActivity.onCreate(MainActivity.java:70)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1047)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2627)
... 11 more
Caused by: java.lang.reflect.InvocationTargetException
at com.gpl.rpg.AndorsTrail.view.StatusView.<init>(StatusView.java:36)
at java.lang.reflect.Constructor.constructNative(Native Method)
at java.lang.reflect.Constructor.newInstance(Constructor.java:446)
at android.view.LayoutInflater.createView(LayoutInflater.java:500)
... 21 more
Caused by: java.lang.NullPointerException
... 25 more




Comments left by users:
Feb 17, 2011 8:11:31 AM
0.6.7
Needs 2 stop force closing an needs rest of map
Feb 14, 2011 4:54:42 PM
0.6.7
It Just Closed!
Feb 12, 2011 9:32:34 PM
0.6.7
Fails on OS application switching
Feb 8, 2011 5:23:28 AM
0.6.7
hihihh
Jan 24, 2011 3:10:33 AM
0.6.7
Locked down with text received
Jan 22, 2011 8:28:00 PM
0.6.7
was ok initially. crashed when trying to open saved game.
Jan 20, 2011 5:36:40 PM
0.6.7
forced closed when I went to open it. I used the home key and held it down to 
open recent apps. I have an Evo 4G froyo
Jan 20, 2011 9:30:12 AM
0.6.7
resume from task switcher. native android.
Jan 16, 2011 1:49:47 PM
0.6.7
Crash after call
Jan 15, 2011 11:52:05 AM
0.6.7
After call, tried to use back button. The had just been taken out of standby.
Jan 14, 2011 12:56:11 PM
0.6.7
crashed in cave
Jan 12, 2011 5:09:04 PM
0.6.7
failed during call
Jan 6, 2011 11:48:36 PM
0.6.7
force quit
Jan 6, 2011 12:06:56 AM
0.6.7
Died after phone call...
Dec 27, 2010 3:27:02 AM
0.6.7
won't let me force close the app
Dec 25, 2010 9:34:52 PM
0.6.7
it locked up
Nov 22, 2010 5:04:30 PM
0.6.4
force close on re open.

```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 24 Feb 2011 at 10:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 10:13
- Changed title: **FC from user reports - StatusView.java:36**

**Comment by GoogleCodeExporter** [comment]
```
Note that this was submitted from players having v0.6.7 , so this bug must have 
been present for quite a while.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 10:16

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved with the changes done for issue 274.
```

Original comment by `oskar.wi...@gmail.com` on 29 Apr 2012 at 3:57
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
